src/content/blog/v0.3_release.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "v0.3 release"
+pubDate: "2025-04-29"
+description: "Blog to accompany v0.3 release"
+draft: true
+author: "AutoEmulate contributors"
+---
+
+We're excited to announce the progress on AutoEmulate in this v0.3.0 release -- we've added several exciting new features advancing the package as an all-purpose emulation toolkit, as well as adding several new tutorials demonstrating the package in action. In the two sections below we look back at "What's new" before looking ahead to "What's next" in the upcoming cycle.
+
+- [What's new]()
+    - [History matching]()
+    - [Dimensionality reduction]()
+    - [Active learning (experimental)]()
+    - [End-to-end workflow example with a user-provided simulator]()
+- [What's next]()
+
+## What's new
+
+### History matching:
+- One of the commonly used techniques in emulation for model calibration is "History Matching". 
+- This technique aims to redefine the parameter space so that the predictions from the emulator are best representative of real observations.
+- This is done by iteratively ruling out regions of the parameter space that are implausible according to observed values.
+
+![image_720](https://hackmd.io/_uploads/ry8dq7T1gx.png)
+
+
+- This [tutorial](https://alan-turing-institute.github.io/autoemulate/tutorials/01_start.html#history-matching) demonstrates how users can employ their own simulator in AutoEmulate's framework. Furthermore it elaborates on the  use of calibration and analysis tools as well adding a visualisation dashboard as a part of AutoEmulate package.
```

Comment:
We removed history matching from the quick start notebook. It's now included in the end-to-end workflow example, which is mentioned as a separate item so it doesn't feel right to include it here to. Maybe we should rearrange the items so that history matching appears above the end-to-end workflow item, which can then reference it.
```suggestion

```